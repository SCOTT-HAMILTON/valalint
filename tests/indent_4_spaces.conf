[Checks]
space_indent = 4

[Result]
errors=24
warnings=0
applied_fixes=24
failed_fixes=0
