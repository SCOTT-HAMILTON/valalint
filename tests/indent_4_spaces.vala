namespace Test {

    /* Toplevel namespace - shift -1 */
    public void foo() {
        warning("Hello");
    }

public void bar(int foo) {
    switch (foo) {
        /* case - shift -1 */
        case 1:
            print(1);
            break;
        /* default - shift -1 */
        default:
            print(0);
            break;
    }

    /* &&, || - shift -1 */
    if (foo == 1
        || foo == 2
        && foo == 3) {
        print();
    } else {
    print();
    }

    printf("foo %d %d",
    1,
    2);
    print(print(
    1, 2));
}

} // namespace Test
