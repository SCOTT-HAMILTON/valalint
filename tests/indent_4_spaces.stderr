build/test/test.vala:3.1-3.5: error: Incorrect identation level 1 (0 expected).
    /* Toplevel namespace - shift -1 */
^^^^^
build/test/test.vala:4.1-4.5: error: Incorrect identation level 1 (0 expected).
    public void foo() {
^^^^^
build/test/test.vala:5.1-5.9: error: Incorrect identation level 2 (1 expected).
        warning("Hello");
^^^^^^^^^
build/test/test.vala:6.1-6.5: error: Incorrect identation level 1 (0 expected).
    }
^^^^^
build/test/test.vala:11.1-11.9: error: Incorrect identation level 2 (1 expected).
        case 1:
^^^^^^^^^
build/test/test.vala:12.1-12.13: error: Incorrect identation level 3 (2 expected).
            print(1);
^^^^^^^^^^^^^
build/test/test.vala:13.1-13.13: error: Incorrect identation level 3 (2 expected).
            break;
^^^^^^^^^^^^^
build/test/test.vala:15.1-15.9: error: Incorrect identation level 2 (1 expected).
        default:
^^^^^^^^^
build/test/test.vala:16.1-16.13: error: Incorrect identation level 3 (2 expected).
            print(0);
^^^^^^^^^^^^^
build/test/test.vala:17.1-17.13: error: Incorrect identation level 3 (2 expected).
            break;
^^^^^^^^^^^^^
build/test/test.vala:22.1-22.9: error: Incorrect identation level 2 (1 expected).
        || foo == 2
^^^^^^^^^
build/test/test.vala:23.1-23.9: error: Incorrect identation level 2 (1 expected).
        && foo == 3) {
^^^^^^^^^
build/test/test.vala:26.1-26.5: error: Incorrect identation level 1 (2 expected).
    print();
^^^^^
build/test/test.vala:30.1-30.5: error: Incorrect identation level 1 (2 expected).
    1,
^^^^^
build/test/test.vala:31.1-31.5: error: Incorrect identation level 1 (2 expected).
    2);
^^^^^
build/test/test.vala:33.1-33.5: error: Incorrect identation level 1 (2 expected).
    1, 2));
^^^^^
build/test/test.vala:36.1-36.5: error: Incorrect identation level 1 (0 expected).
    /*
^^^^^
build/test/test.vala:37.1-37.6: error: Incorrect identation level 1 (0 expected).
     * Multiline comment.
^^^^^^
build/test/test.vala:38.1-38.6: error: Incorrect identation level 1 (0 expected).
     */
^^^^^^
build/test/test.vala:39.1-39.5: error: Incorrect identation level 1 (0 expected).
    public void baz() {
^^^^^
build/test/test.vala:40.1-40.5: error: Incorrect identation level 1 (0 expected).
    }
^^^^^
